app/components/contributor-list/template.hbs

```diff
@@ -13,7 +13,7 @@
     <tr class="metadata {{if contributor.isCurrentUser 'current-user'}} {{if contributor.showMetadata 'visible'}}">
       <td colspan="2">
         <ul>
-          <li><a href="https://testnet.etherscan.io/address/{{contributor.address}}">Inspect Ethereum transactions</a></li>
+          <li><a href="https://testnet.etherscan.io/address/{{contributor.account}}">Inspect Ethereum transactions</a></li>
```

Comment:
I don't fully understand why the address is called account now, because didn't we allow to change an address, specifically so that it is not the same as an account?

Reply:
the change of an address is still supported (see the contracts) 
the word `address` is reserved in solidity - it is the type. that's why we named the attribute `account` in the contract. To be consistent with the smart contract I changed it in the frontend also to to account and I think we should always use the word account.

Reply:
Ah ok. I find it a bit misleading as a term for a single property of an account/person, but better that it's the same on both sides for sure.

Reply:
it is account because ethereum talks about accounts - and it is the ethereum account of that contributor. - but ethereumAccount is a bit long.

Reply:
Makes sense.